localweather.js

```diff
@@ -0,0 +1,45 @@
+var API_KEY = "b66507bb73d941dcbcc192051170507";
+var cel = false;
+var wd;
+
+
+function switchUnit(currentTempF, c) {
+    if (c) {
+        var valueInC = (currentTempF - 32) * (5 / 9) + '&deg;' + 'C';

```

Comment:
moglas zrobic funcje pomocnicza "toDegrees()", ktora bralaby wartosc numeryczna + stopien i laczyla to ladnie w string